docs/documentation_routes.js

```diff
@@ -0,0 +1,78 @@
+var express = require('express'),
+    fs      = require('fs'),
+    marked  = require('marked'),
+    path    = require('path'),
+    router  = express.Router(),
+    utils   = require(__dirname + '/../lib/utils.js');
+
+
+// Page routes
+
+// Docs index
+router.get('/', function (req, res) {
+  
+  res.render('index');
+
+});
+
+router.get('/install', function (req, res) {
+  console.log('hi');
+  url = utils.getLatestRelease();
+  res.render('install', { 'releaseURL' : url });
+});
+
+// Pages in install folder are markdown
+router.get('/install/:page', function (req, res) {
+  redirectMarkdown(req.params.page, res);
+  var doc = fs.readFileSync(path.join(__dirname + '/documentation/install/' + req.params.page + ".md"), "utf8");
```

Comment:
I think this would be more readable if broken out into 2 or 3 lines eg `var filepath= path.join ...`